### DISCUSSION TITLE
Transparent Background

### DISCUSSION THREAD

**Discussion by GenieDesu** [bnpr/Malt#31] [Q&A] [answered]
Is there a way to render with a transparent background?

**Comment by GenieDesu** [comment]
And have any Idea how to change the background color?

**Comment by pragma37** [comment, marked as answer]
Hey! Sorry for the super late reply. I didn't got any notification.

The background color can be changed from the Scene properties panel.
Transparent backgrounds are not yet supported but it's high on my priority list.

**Comment by GenieDesu** [comment]
Hi @pragma37 I'm back again after a couple of years, so do we have a transparent feature now?

**Reply by pragma37** [reply]
Sure!
Just look for the Background Node Tree and change the Background alpha:
![imagen](https://github.com/bnpr/Malt/assets/36610999/7cad71e2-727b-4ad7-af7b-9e37035d6b03)

Sorry for the late reply!

**Comment by Dojoe721** [comment]
is it supported yet?

**Reply by HSDSZ** [reply]
supported long time ago. Just change the Background Node Tree
